### PR TITLE
Create workflow for dev releases from develop branch

### PR DESCRIPTION
**Pull request by TheLastCicada**

### CODE REVIEW

**Review by cmmarslender — APPROVED** [review]

File: .github/workflows/auto-release-rc.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  group: rc-release-check
+
+jobs:
+  release-dev:
+    name: Release rc version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clean workspace
+        uses: Chia-Network/actions/clean-workspace@main
+
```
Comment (cmmarslender) [inline .github/workflows/auto-release-rc.yml:21]:
```suggestion
```

Shouldn't need to use this anymore - all handled by our runners automatically now

File: .github/workflows/build-installers.yaml [hunk 2]
```diff
@@ -258,32 +290,60 @@ jobs:
...
+          prerelease: ${{steps.is_latest.outputs.IS_PRERELEASE}}
+          make_latest: "${{steps.is_latest.outputs.IS_LATEST}}"
           files: |
             ${{ env.DMG_FILE }}
             ${{ env.DEB_FILE }}
             ${{ env.EXE_FILE }}
             ${{ env.WEB_FILE }}
 
       - name: Get repo name
+        if: startsWith(github.ref, 'refs/tags/') && !contains( github.ref, '-rc')
```
Comment (cmmarslender) [inline .github/workflows/build-installers.yaml:331]: Do you need to also exclude 'dev' here, or are you removing dev completely
If removing dev completely, can probably remove from the "is latest" logic above too?

**Review by cmmarslender — CHANGES_REQUESTED** [review]

File: .github/workflows/build-installers.yaml (hunk 2 above)
Comment (TheLastCicada) [reply .github/workflows/build-installers.yaml:331]: Good point - I was supporting both methods while I was working on this, but don't need the dev stuff anymore, I'll look at what I can remove related to that.

Comment (TheLastCicada) [reply .github/workflows/build-installers.yaml:327]: Removed

**Pull request merged**